.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,29 @@
+name: Integration Tests
+
+on:
+  push:
+      branches:
+      - main
+  pull_request:
+      branches:
+      - main
+  workflow_dispatch:
+
+permissions:
+  # Necessary to write the branch
+  contents: write
```

Comment:
Its part of https://github.com/open-component-model/ocm-integrationtest/pull/31/files#diff-60c6c2c5210083aa87c9834799112e5f8ab1252b454f90c8f5c93028943ba60dR41 but this I can only fix after rewriting the integration test some more

Basically one kind of run (that can never be triggered from here) needs to push the test results into the repo. However because github actions forces me to write this permission when it deviates from default, I also have to write it here.

Since eventually everything disappears in terms of other triggers, there will be no need to keep this and then I can remove it